analyses/transcriptomic-dimension-reduction/01-transcriptomic-analysis-prep.Rmd

```diff
@@ -0,0 +1,316 @@
+---
+title: "Unsupervised Analysis of Transcriptomic Differences - Data Prep"
+author: Chante Bethell for CCDL 2019
+output:
+  html_notebook:
+    toc: true
+    toc_float: true
+---
+
+This notebook is the first of two notebooks that demonstrate samples in the PBTA cluster by cancer type using dimensionality reduction techniques, namely, 
+[Principal Component Analysis](https://www.rdocumentation.org/packages/stats/versions/3.6.0/topics/prcomp) (PCA), 
+[t-Distributed Stochastic Neighbor Embedding](https://cran.r-project.org/web/packages/Rtsne/Rtsne.pdf) (t-SNE), 
+and [Uniform Manifold Approximation and Projection](https://cran.r-project.org/web/packages/umap/vignettes/umap.html) (UMAP).
+
+This notebook focuses on the preparation of the expression data and the execution of the dimension reduction techniques. 
+
+It addresses [issue #9](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/9)
+in the Open-PBTA analysis repository. 
+
+## Output files:
+- `analysis/transcriptomic-dimension-reduction/results/15.17_PCA_scores.tsv`
+- `analysis/transcriptomic-dimension-reduction/results/15.17_t-SNE_scores.tsv`
+- `analysis/transcriptomic-dimension-reduction/results/15.17_UMAP_scores.tsv`
+- `analysis/transcriptomic-dimension-reduction/results/109.299_PCA_scores.tsv`
+- `analysis/transcriptomic-dimension-reduction/results/109.299_t-SNE_scores.tsv`
+- `analysis/transcriptomic-dimension-reduction/results/109.299_UMAP_scores.tsv`
+- `analysis/transcriptomic-dimension-reduction/results/rsem_pca_model.RDS`
+- `analysis/transcriptomic-dimension-reduction/results/rsem_tsne_model.RDS`
+- `analysis/transcriptomic-dimension-reduction/results/rsem_umap_model.RDS`
+- `analysis/transcriptomic-dimension-reduction/results/kallisto_pca_model.RDS`
+- `analysis/transcriptomic-dimension-reduction/results/kallisto_tsne_model.RDS`
+- `analysis/transcriptomic-dimension-reduction/results/kallisto_umap_model.RDS`
+
+_Note that the tsv files holding the dimension reduction scores represent the scores for the RSEM (denoted by the output tsv files beginning with 15.17) and Kallisto expression data (denoted by the output tsv files beginning with 109.299)._
```

Comment:
Why is naming the files this way rather than using `rsem` or `kallisto` in the filenames preferable?

Reply:
The naming is a result of the `as.name` function. This was meant to be a temporary fix, until I find a better method of using the `transposed_expression_matrix` argument for naming.